common/transform/src/main/java/ai/h2o/mojos/deploy/common/transform/MojoScorer.java

```diff
@@ -57,6 +57,7 @@ public class MojoScorer {
   private final ContributionRequestToMojoFrameConverter contributionRequestConverter;
   private final MojoPipelineToModelInfoConverter modelInfoConverter;
   private final CsvToMojoFrameConverter csvConverter;
+  private final ScoreRequestTransformer scoreRequestTransformer = new ScoreRequestTransformer();
```

Comment:
you are absolutely right, initial i trying to minimize the scope of changes so to leave the constructor unchanged since it is also used in different libraries but it is not a good practice, updated https://github.com/h2oai/dai-deployment-templates/pull/327/commits/844043a958779dce72c770bd3d8e72d72ea41959